test/cache-tests.js

```diff
@@ -71,7 +71,7 @@ function basicTests(factory) {
 
     it('should store and retrieve binary buffers', () => {
         const cache = factory();
-        const buffer = new Buffer(2 * 1024 * 1024);
+        const buffer = Buffer.alloc(2 * 1024 * 1024);
```

Comment:
I don't understand why any of these changes are needed, can you explain why they are needed?

Reply:
https://nodejs.org/api/buffer.html#buffer_buffer_from_buffer_alloc_and_buffer_allocunsafe